Testscripts/Windows/NTTTCP-MONITOR-THROUGHPUT.ps1

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
```

Comment:
will this script keep monitoring the status, or just start the test?

Reply:
yes, only start the test.  see PR description.